llvm/unittests/Support/SuffixTreeTest.cpp

```diff
@@ -140,4 +140,138 @@ TEST(SuffixTreeTest, TestExclusion) {
   }
 }
 
+// Tests that the SuffixTree is able to find the following substrings:
+// {1, 1} at indices 0, 1, 2, 3, and 4;
+// {1, 1, 1} at indices 0, 1, 2, and 3;
+// {1, 1, 1, 1}  at indices 0, 1, and 2; and
+// {1, 1, 1, 1, 1} at indices 0 and 1.
+//
+// This is a FIX to the Test TestSingleCharacterRepeat
```

Comment:
This is the fix for TestSingleCharacterRepeat.

Can you also insert these 3 tests right after TestSingleCharacterRepeat so that the related tests are placed back-to-back?

Reply:
Will do!